bench/config/templates/Procfile

```diff
@@ -2,15 +2,18 @@
 redis_cache: redis-server config/redis_cache.conf
 redis_queue: redis-server config/redis_queue.conf
 {% endif %}
-web: bench serve {% if webserver_port -%} --port {{ webserver_port }} {%- endif %}
-
+{% if not skip_web %}
+web: bench serve {% if with_coverage -%} --with-coverage  {%- endif %} {% if webserver_port -%} --port {{ webserver_port }} {%- endif %}
```

Comment:
Any cases where webserver_port was null here?

Reply:
This was in there before, but I've not personally observed a case where this would run on `80`